### DISCUSSION TITLE
Anthropic Claude 3 haiku missing

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#2185] [Feature Requests & Suggestions]
Anthropic Claude 3 haiku was released a few days ago. But it not present in LibreChat. I thought the app fetches all available models automatically but that doesn't seem to be the case here

**Comment by danny-avila** [comment]
> Anthropic Claude 3 haiku was released a few days ago. But it not present in LibreChat. I thought the app fetches all available models automatically but that doesn't seem to be the case here

Your post is timely because I just added it: https://github.com/danny-avila/LibreChat/pull/2184

Also I could not find any resource on fetching available models from Anthropic, I am fairly certain it's not possible with their API, so we don't fetch them, and instead maintain a default list.

You can also override/customize the models list at any time with the environment variable

```.env
ANTHROPIC_MODELS=claude-3-opus-20240229,claude-3-sonnet-20240229,claude-3-haiku-20240307
```
